<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messageWindow</class>
 <widget class="QDialog" name="messageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="mainArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255, 0.9);
border-radius: 30px;</string>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font-family: 'MiSans Demibold';
font-size: 25px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="QLabel" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>501</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
font-family:'MiSans';
line-height: 50px;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgba(195, 195, 195, 0.5);
font-family: 'MiSans';</string>
    </property>
    <property name="text">
     <string>чбо  шод</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
